source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,321 @@ Sorts Builders
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts <apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?>`
+type. 
+
+The following examples show you how to use the static factory
+methods provided by the ``Sorts`` class to sort your queries.
+The examples use a sample collection ``sort_example`` that contains
+the following documents:
+
+.. code-block:: none
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Data
+------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
+with ``Sorts.ascending()``, and data can be sorted from largest to smallest with
+``Sorts.descending()``. The following two subsections provide examples
+on how to use these two methods. 
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()``
+static factory method. An ascending sort returns 
+your documents from smallest to largest on the field name specified.
+
+The ``ascending`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(Sorts.ascending("<field name>"))
+
+This will return a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection sorted 
+from smallest to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use ``ascending`` to
+   sort the ``sort_example`` collection by ``"_id"``:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson id_sort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(id_sort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the above code snippet should look something like this: 
+   
+   .. code-block:: none
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+   
+An important thing for you to keep in mind is that MongoDB does
```

Comment:
Let's try to keep sentences concise by avoiding/truncating unnecessary phrases like "an important thing for you to keep in mind".

Reply:
"Note That", "Keep in mind", short phrases from sync review. "Imporant thing" is a generic, vague, and generally unhelpful term.

Reply:
Thanks for catching this, when it is pointed out I can see that this "An important thing for you to keep in mind" is not performing any real work in the sentence. Fix: https://github.com/mongodb/docs-java/pull/59/commits/607668630503604b9749b5f1856514bd2470b027

Going forward, I will try to improve my sentence revision sense to catch these sorts of unnecessary phrases before review.